### DISCUSSION TITLE
[2021.08.26] 회사에 있는 사람

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#56] [Algorithm]
[회사에 있는 사람](https://www.acmicpc.net/problem/7785)

**Comment by pyunji** [comment]
[회사에 있는 사람 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/BOJ/BOJ_7785.java)

**Comment by hyk339** [comment]
[회사에 있는 사람 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/commit/4c18e53d60a947f5fb783cda951616e6414d27eb)

**Comment by chaeyeon4u** [comment]
[회사에 있는 사람 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_32669738.java)

**Comment by leechang4** [comment]
[회사에 있는 사람 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/BOJ/BOJ%207785.java)
